File: Python3/프로그래머스/2/42578. 의상/의상.py
def solution(clothes):
    dict = {}
    for cloth, category in clothes:
        if category in dict:
            dict[category] += 1
        else:
            dict[category] = 1
    
    answer = 1
    for key in dict.keys():
        answer *= (dict[key] + 1)
        
    answer -= 1
    return answer

File: C++/프로그래머스/2/42578. 의상/의상.cpp
#include <string>
#include <vector>
#include <bits/stdc++.h>

using namespace std;

int solution(vector<vector<string>> clothes) {
    int answer = 1;
    map<string, int> maps;
    
    for (auto& c : clothes) {
        maps[c[1]]++;
    }
    
    for (auto& [key, value] : maps) {
        cout << value << "\n";
        answer *= (value + 1);
    }
    cout << answer << "answer";
    answer--;
    
    return answer;
}